games.yaml

```diff
@@ -687,9 +687,9 @@
 
 - name: [Cannon Fodder, Cannon Fodder (series)]
   clones:
-    - name: OpenFodder
+    - name: Open Fodder
       url: https://github.com/segrax/openfodder
-      info: development halted, resources are loaded / maps drawn, C++
+      info: Active Development, Intro & Recruitment Screen Done, C++
```

Comment:
Can you please de-capitalize this to match style of descriptions everywhere else?